README.md

```diff
@@ -1,7 +1,42 @@
-# ARQO
+# ARQO ![ARQO](docs/images/logo.png)
 
 ARQO (Active Record Query Objects) is a minimal gem that let you use Query Objects in an easy and Rails friendly way. It leverages `ActiveRecord` features and tries to make query objects as intuitive as possible for developers. In combination with the documentation we hope `ARQO` helps people keep their projects well structured and healthy.
 
+![CI](https://github.com/rootstrap/arqo/workflows/ci/badge.svg)
+[![Maintainability](https://api.codeclimate.com/v1/badges/5ed2b32bfdf09746bd82/maintainability)](https://codeclimate.com/github/rootstrap/arqo/maintainability)
+[![Test Coverage](https://api.codeclimate.com/v1/badges/5ed2b32bfdf09746bd82/test_coverage)](https://codeclimate.com/github/rootstrap/arqo/test_coverage)
+
+## Table of Contents
+
+- [Motivation](#motivation)
+  - [Why ARQO?](#why-arqo)
+- [Installation](#installation)
+- [Usage](#usage)
+  - [Setting up a query object](#setting-up-a-query-object)
+  - [Deriving the model](#deriving-the-model)
+  - [Chaining scopes](#chaining-scopes)
+- [Development](#development)
+- [Contributing](#contributing)
+- [License](#license)
+- [Code of Conduct](#code-of-conduct)
+- [Credits](#credits)
+
+## Motivation
+
+`ActiveRecord` provides us with an amazing abstraction of the database structure, allowing us to form queries in a simple way. Unfortunately models can grow big for several reasons, one of them being adding a lot of scopes or placing querying logic in methods.
```

Comment:
`allowing us to form queries` --> `allowing us to write queries` ?

Reply:
`Unfortunately models can grow big` --> `Unfortunately models can grow/get fat` ?

Reply:
I don't like using the word fat, but I changed it for grow large